recipe/meta.yaml

```diff
@@ -11,21 +11,30 @@ source:
   sha256: ff3b5a1ac0010d0967869f723e687d42cabc7dccf33b14934c92aa5168d260b3
 
 build:
-  number: 2
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
   skip: True  # [not win]
+  skip: True  # [py<38]
```

Comment:
Just curious - is it correct that there are two `skip` lines?

Reply:
Yeah, after the symbol `#` are selectors.
But we can also use` skip: True  # [not win or py<38]`